backend/pom.xml

```diff
@@ -25,12 +25,26 @@
 		<liquibase.version>3.8.0</liquibase.version>
 		<prometheus.version>0.8.1</prometheus.version>
 		<protobuf.version>3.8.0</protobuf.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <aws.sdk.version>1.11.106</aws.sdk.version>
```

Comment:
this is unused?

Reply:
seems so... I was using a prebuilt minio example from another project and inadvertently transferred over more than was needed it seems. I've also removed this now